### DISCUSSION TITLE
DateRangePicker

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#3686] [Q&A]
In my business logic there is "statement" business entity. 
Statement starts on some day of week and its duration is always multiple of 7 (In weeks)

Lets pretend statements always start on Monday, in javascript monday is 1, and its duration is 7 days (1 week).
So i need DateRangePicker to only allow user to choose mondays of each week.
Here is how i implemented it:
https://codesandbox.io/p/sandbox/upbeat-ganguly-qqlqfx?file=%2Fsrc%2FApp.tsx

In codesandbox i provided, you can choose date range from 4 of march which is Monday, to 18 of March which is also monday.
**BUT** because days from 5 to 17 of March are disabled, i can not press OK button and submit the date.
Please help me what can i do with this.

**Comment by simonguo** [comment]
> https://codesandbox.io/p/sandbox/upbeat-ganguly-qqlqfx?file=%2Fsrc%2FApp.tsx

Non-public example.
![image](https://github.com/rsuite/rsuite/assets/1203827/5e99235b-087f-410d-9d0c-3d5808bb31a7)


I guess choosing the whole week's feature should be what you want.

```tsx
<DateRangePicker hoverRange="week" isoWeek ranges={[]}  />
<DateRangePicker hoverRange="week" isoWeek oneTap ranges={[]}  />
```
> https://rsuitejs.com/components/date-range-picker/#select-whole-week-whole-month
